site/docs/0.0/components/navbar-off-canvas.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: docs
+title: Navbar Off-Canvas
+description: Documentation and examples for Arizona Bootstrap's main menu navigation component
+group: components
+toc: true
+---
+
+### Off-Canvas Mobile Nav
+
+Adding the `.navbar-off-canvas` class to your `<nav>` will incorporate an off-canvas experience for mobile devices.
+
+{% capture example %}
+<div class="d-lg-none d-flex justify-content-end justify-content-lg-start">
+  <button data-toggle="offcanvas" type="button" data-target="#navbarOffcanvasDemo" aria-controls="navbarOffcanvasDemo" class="btn btn-red">
+    <span class="material-icons"> menu </span>
+  </button>
+</div>
+<nav class="navbar-off-canvas" id="navbarOffcanvasDemo">
```

Comment:
I might be in favor of changing the `.navbar-off-canvas` class to `.navbar-offcanvas` so it matches the other classes, like `.navbar-offcanvas-header`. Just for the sake of consistency. (that was my bad)